components/admin_console/billing/billing_subscriptions/limit_reached_banner.test.tsx

```diff
@@ -65,7 +65,7 @@ const base = {
     recurring_interval: '',
 };
 
-const starter = {...base, sku: CloudProducts.STARTER};
+const free = {...base, sku: CloudProducts.STARTER};
```

Comment:
Not a blocker, but since this variable name has also changed, we can also change `CloudProducts.STARTER` to `CloudProducts.FREE`

Reply:
We're going to keep for now the sku name as cloud-starter, and this variable is more related to the sku name than product name. I see the reason behind this change, at first I did it.

Reply:
Alright. Got it.